### ISSUE TITLE
enunciate-2.0.0.xsd scheme is not valid

### ISSUE THREAD

**Issue by jskovjyskebankdk** [stoicflame/enunciate#199]
Eclipse complains about it (line 104 and 119)

And so does the page http://www.utilities-online.info/xsdvalidation/ (it also does not like line 229)

I must admit to not knowing enough about schemas to help fix it. Sorry. But report it, I can :)

```
Not valid.
Error - Line 104, 38: org.xml.sax.SAXParseException; lineNumber: 104; columnNumber: 38; src-ct.1: Complex Type Definition Representation Error for type 'description'.  When <complexContent> is used, the base type must be a complexType. 'string' is a simpleType.
Error - Line 119, 38: org.xml.sax.SAXParseException; lineNumber: 119; columnNumber: 38; src-ct.1: Complex Type Definition Representation Error for type 'license'.  When <complexContent> is used, the base type must be a complexType. 'string' is a simpleType.

Error - Line 229, 34: org.xml.sax.SAXParseException; lineNumber: 229; columnNumber: 34; cos-nonambig: c-xml-client and WC[##any] (or elements from their substitution group) violate "Unique Particle Attribution". During validation against this schema, ambiguity would be created for those two particles.
```

**Comment by stoicflame** [comment]
Fixed at 5cd9b30. The error on 229 happens because the XSD validator isn't applying schema 1.1; it's just applying 1.0.

**Comment by stoicflame** [comment]
[Enunciate 2.1 has been released.](https://github.com/stoicflame/enunciate/releases/tag/v2.1.0)
